das-dashboard/backend/shared/exceptions/exception_handlers.py

```
@@ -189,4 +181,29 @@ async def handle_custom_value_error(
+    ):
+
+        return JSONResponse(
+            status_code=500,
+            content={
+                "message": exc.message
+            }
+        )
+
+    async def handle_config_load_error(
+            self,
+            request: Request,
```

Ruff (0.15.21)

[warning] Unused method argument: `request`

(ARG002)